dev-env/windows/libexec/install.ps1

```diff
@@ -15,7 +15,10 @@ function da_install {
     }
 
     da_enable_scoop
-    $isScoopEnabled = da_is_scoop_installed
+    # $isScoopEnabled = da_is_scoop_installed
+    # Temporary because we upgraded scoop so we need to force a reinstall. After all
+    # machines have been refreshed we can remove this.
+    $isScoopEnabled = $false
```

Comment:
This is stupid but we can kill it tomorrow and given that everyone is blocked on this, I don’t want to spend the time to figure out a nicer solution.